src/showcase/app/business/business-examples/status-examples/status-with-message-example/status-with-message-example.component.html

```diff
@@ -0,0 +1,6 @@
+<h4>Valid</h4>
+<sbb-status type="valid" message="Example message"></sbb-status>
+<h4>Warning</h4>
+<sbb-status type="warning" message="Example message"></sbb-status>
+<h4>Invalid</h4>
+<sbb-status type="invalid" message="Example message"></sbb-status>
```

Comment:
```suggestion
<sbb-status type="invalid" message="Service is down"></sbb-status>
```

Reply:
There could be other message but I think it's better to not only use "Example message" in all status.